### ISSUE TITLE
Refactor Calculations

### ISSUE THREAD

**Issue by jonLicari** [jonLicari/finance-report#28]
# TODO
- Move calculations to separate file (BLOCKED by calculations refactor)

_Originally posted by @jonLicari in https://github.com/jonLicari/finance-report/issues/26#issuecomment-1926277208_

**Comment by jonLicari** [comment]
# Changes
- store cashflow totals in an Expense object
- [x] process cashflow transactions on a monthly basis
- [x] produce a YTD object from the monthlies to simplify calculations and plots

**Comment by jonLicari** [comment]
Need to refactor category and subcategory totals in new PR since it was out of scope of the main calculations refactor.
Changes will be tracked by PR Refactor Category Calculations #31

**Comment by jonLicari** [comment]
Reopening since refactoring of category calculations needs to close out first.
